README.md

```diff
@@ -0,0 +1,5 @@
+# Minecraft (Flatpak)
+
+#### Resource Packs, Mods etc
+
+You can install these to `~/.var/app/com.mojang.Minecraft/data`.
```

Comment:
Is this correct? I see that Minecraft is installed in `~/.var/app/com.mojang.Minecraft/.minecraft`.

Reply:
Oops, I am dumb, ignore my previous comment. :) Anyway, I am not sure if it is a good idea to put Minecraft directly to `$XDG_DATA_HOME`. `$XDG_DATA_HOME/minecraft` (for example) would be better, imho.